### PR TITLE
Feature/faq

### PR DESCRIPTION
**Pull request by Klemmensen**
## Description

- Adds a new FAQ page

## References

https://github.com/Energinet-DataHub/energy-origin-issues/issues/420

### CODE REVIEW

File: libs/eo/faq/routing/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# FAQ page routing
```
Comment (LayZeeDK) [inline libs/eo/faq/routing/README.md:1]: nitpick: A domain can have more than one *page*. Avoid using the word *page*.
```suggestion
# FAQ routing
```

File: libs/eo/faq/shell/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# EnergyOrigin FAQ page shell
```
Comment (LayZeeDK) [inline libs/eo/faq/shell/README.md:1]: nitpick: A domain can have more than one *page*. Avoid using the word *page*.
```suggestion
# EnergyOrigin FAQ shell
```

File: libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    </p>
+
+    <h3 #where_does_the_data_come_from>Where does the data come from?</h3>
+    <p>
+      All data related to metering points, consumption and production is served
+      by the Energinet DataHub. Data related to the origin of energy and
+      corresponding emissions stems from Energi Data Service.
+    </p>
+
+    <h3 #how_can_i_influence_the_development>
```
Comment (LayZeeDK) [inline libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts:81]: issue: use *camelCase* for template reference variables.
```suggestion
    <h3 #howCanIInfluenceTheDevelopment>
```

File: libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+      electricity declaration and corresponding emissions for 2021.
+    </p>
+
+    <h3 #who_can_access_the_platform>Who can access the platform?</h3>
+    <p>
+      Currently, the platform only offers company login via NemID. Later on, it
+      will be possible for private individuals to login via NemID / MitID.
+    </p>
+
+    <h3 #where_does_the_data_come_from>Where does the data come from?</h3>
```
Comment (LayZeeDK) [inline libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts:74]: suggestion: To take accessibility and usability into account, we should also use fragments (IDs) to support linking and navigating by keyboard to the FAQ items.
```suggestion
    <h3 #where_does_the_data_come_from id="where-does-the-data-come-from">Where does the data come from?</h3>
```

File: libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+      >Origins of energy | Energinet</a
+    >
+  `,
+})
+export class EoFaqShellComponent {
+  scrollToTargetElement(targetElement: HTMLHeadingElement): void {
+    targetElement.scrollIntoView({
+      behavior: 'smooth',
+      block: 'start',
+    });
```
Comment (LayZeeDK) [inline libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts:107]: suggestion: FYI, this is going to be [an issue for accessibility](https://css-tricks.com/smooth-scrolling-accessibility/).

File: libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    </h3>
+    <p>
+      You are more than welcome to participate in our LinkedIn user group, a
+      digital forum for users, in which we post sketches, questions and gather
+      ideas and suggestions for new development and improvements. It’s
+      non-binding and you can either just follow along without getting involved
+      or comment when you have time.
+    </p>
+
+    <h3 #where_can_i_read_more>Where can I read more?</h3>
```
Comment (LayZeeDK) [inline libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts:92]: suggestion: Did you consider using a directive to dynamically apply a fragment (element ID) based on the headline text and to add smooth scroll support?

File: libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'eo-faq-shell',
+  styles: [
+    `
+      :host {
+        display: block;
+        width: calc(200 * var(--watt-space-xs));
```
Comment (LayZeeDK) [inline libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts:26]: suggestion: Say it like it is.
```suggestion
        width: 800px; // Magic number by designer
```

File: libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+      non-binding and you can either just follow along without getting involved
+      or comment when you have time.
+    </p>
+
+    <h3 #where_can_i_read_more>Where can I read more?</h3>
+    <p>
+      You can read more about the history and intention of Energy Origin on our
+      website:
+    </p>
+    <a href="https://energinet.dk" target="_blank"
```
Comment (LayZeeDK) [inline libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts:97]: issue: This is not a URL to information about Energy Origin.

File: libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.spec.ts [hunk 9]
```diff
@@ -161,13 +161,13 @@ describe(WattNavListModule.name, () => {
     });
   });
 
-  describe('Verify links added with href attribute opens in expected windows', () => {
+  describe(`${WattNavListModule.name} - Verify links added with href attribute opens in expected windows`, () => {
```
Comment (LayZeeDK) [inline libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.spec.ts:164]: suggestion: Revert this change.
```suggestion
  describe('Verify links added with href attribute opens in expected windows', () => {
```

File: tsconfig.base.json [hunk 10]
```diff
@@ -111,6 +111,7 @@
       "@energinet-datahub/dh/shared/ui-date-time": [
         "libs/dh/shared/ui-date-time/src/index.ts"
       ],
+      "@energinet-datahub/eo-faq/routing": ["libs/eo/faq/routing/src/index.ts"],
```
Comment (LayZeeDK) [inline tsconfig.base.json:114]: issue: This import path alias doesn't follow our format. Make sure to update all existing import statements using this library.
```suggestion
      "@energinet-datahub/eo/faq/routing": ["libs/eo/faq/routing/src/index.ts"],
```

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

File: libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts (hunk 8 above)
Comment (Klemmensen) [reply libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts:97]: Got a more specific link from Tina, PO - Pointing to: https://en.energinet.dk/Electricity/DataHub/Energy-Origin

File: libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts (hunk 5 above)
Comment (Klemmensen) [reply libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts]: Enabled keyboard navigation by using anchors

**Review by LayZeeDK — DISMISSED** [review]

**Review by LayZeeDK — APPROVED** [review]
